IBM/ibm-aix/ingest/parser.yml

```diff
@@ -83,5 +83,6 @@ stages:
   set_command_line_fields:
     actions:
       - set:
-          process.command_line: '{{parsed_event.message.process_name}} {{parsed_event.message.process_command_line}}'
-        filter: '{{parsed_event.message.process_name != None and parsed_event.message.process_command_line != None}}'
+          process.command_line: '{{parsed_event.message.process_name}} {{parsed_event.message.file_name}} {{parsed_event.message.process_command_line}}'
+        filter: 
+          '{{parsed_event.message.process_name != None and parsed_event.message.process_command_line != None}}'
```

Comment:
```suggestion
        filter: '{{parsed_event.message.process_name != None and parsed_event.message.process_command_line != None and parsed_event.message.file_name !=  None }}'
```

Reply:
This filter is partially wrong: if any value is `None` then you will be missing the resulting field

ping @squioc if you have any idea on this one